app/hotels/src/search/SearchQueryHelpers.js

```diff
@@ -40,7 +40,7 @@ export const getSearchQueryParams = (
 
 export const isDateBeforeToday = (date: Date) => {
   const today = new Date();
-  return date < today && date.getUTCDay() !== today.getUTCDay();
```

Comment:
getUTCDay() returns the number of the day, i.e. tuesday is always 2, so for past date that was same week day, it would return wrong result.